### PR TITLE
[refactor][tools][sc-server] Restruct logs in sc-server

### PR DESCRIPTION
**Pull request by NikitaZotov**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/CONTRIBUTING.md)
* [x] Update changelog
* [ ] Update documentation

### CODE REVIEW

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
There is no `components`. Rename according to the Standard
image
Make the last log message to understand that all is correct and ready to use

File: sc-tools/sc-server/src/sc-server-impl/sc_server.cpp
[the remark this answers is not in the document]
Comment (NikitaZotov) [reply sc-tools/sc-server/src/sc-server-impl/sc_server.cpp:1]: There are library logs. We don't extend it, so I set such logs by default

File: sc-tools/sc-server/src/sc-server-impl/sc_server.cpp [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+  if (m_ioThread.joinable())
+  {
+    LogMessage(ScServerLogMessages::app, "Close input-output processing");
+
+    for (auto & it : *m_connections)
+    {
+      try
+      {
+        m_instance->close(it, websocketpp::close::status::normal, "I finish work");
```
[the remark this answers is not in the document]
Comment (NikitaZotov) [reply sc-tools/sc-server/src/sc-server-impl/sc_server.cpp:90]: sc-server

**Comment by NikitaZotov** [comment]
> There is no `components`. Rename according to the Standard image Make the last log message to understand that all is correct and ready to use

there are no about how to write logs in sc-console in the Standart

File: sc-tools/sc-server/src/sc-server-impl/sc_server.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply sc-tools/sc-server/src/sc-server-impl/sc_server.cpp:91]: ну так write sc-server please

File: sc-tools/sc-server/src/main.cpp [hunk 2]
```diff
@@ -30,6 +30,47 @@ void PrintStartMessage()
...
+  sc_bool status = SC_FALSE;
+  try
+  {
+    server = ScServerFactory::ConfigureScServer(serverParams, memoryConfig.GetParams());
+    server->Run();
+    server->ResetLogger(ScServerFactory::ConfigureScServerLogger(server, serverParams));
+
+    status = SC_TRUE;
+  }
+  catch (utils::ScException const & e)
```
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply sc-tools/sc-server/src/main.cpp:44]: What does your smile mean? Lets doiscuss

File: sc-tools/sc-server/src/sc-server-impl/sc-memory-json/sc-memory-json-event/sc_memory_json_events_handler.hpp [hunk 3]
```diff
@@ -47,8 +47,12 @@ class ScMemoryJsonEventsHandler : public ScMemoryJsonHandler
       sc_bool & isEvent) override;
 
   ScMemoryJsonPayload HandleCreate(
-      ScServerConnectionHandle const & hdl, ScMemoryJsonPayload const & message, ScMemoryJsonPayload & errorsPayload);
+      ScServerConnectionHandle const & hdl,
```
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply sc-tools/sc-server/src/sc-server-impl/sc-memory-json/sc-memory-json-event/sc_memory_json_events_handler.hpp:50]: What does your smile mean? Lets doiscuss

File: sc-tools/sc-server/src/sc-server-impl/sc_server.hpp [hunk 4]
```diff
@@ -11,193 +11,48 @@
...
-
-    LogMessage(ScServerLogMessages::app, "Shutdown sc-server");
-
-    delete m_instance;
-    m_instance = nullptr;
-
-    delete m_log;
-    m_log = nullptr;
-  }
+  explicit ScServer(std::string hostName, size_t port, sc_memory_params params);
```
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply sc-tools/sc-server/src/sc-server-impl/sc_server.hpp:21]: What does your smile mean? Lets doiscuss

File: sc-tools/sc-server/src/sc-server-impl/sc_server.cpp [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+  LogMessage(ScServerLogMessages::app, "Start lookup memory state");
+  m_updateStatisticsThread = std::thread(&ScServer::UpdateMemoryStatistics, &*this);
+  m_saveMemoryThread = std::thread(&ScServer::SaveMemory, &*this);
+
+  LogMessage(ScServerLogMessages::app, "Start actions processing");
+  m_actionsThread = std::thread(&ScServer::EmitActions, &*this);
+
+  LogMessage(ScServerLogMessages::app, "Start input-output processing");
+  m_ioThread = std::thread(&ScServerCore::run, &*m_instance);
```
[the remark this answers is not in the document]
Comment (MksmOrlov) [reply sc-tools/sc-server/src/sc-server-impl/sc_server.cpp:60]: Why "Close input-output processing", not End? 
If you write _"start"_, write _"end"_, if you write "close" write "open"

**Comment by MksmOrlov** [comment]
> > There is no `components`. Rename according to the Standard image Make the last log message to understand that all is correct and ready to use
> 
> there are no about how to write logs in sc-console in the Standart

but there is _yes_ about what is a component

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
Check unresolved

File: sc-tools/sc-server/src/sc-server-impl/sc_server.hpp (hunk 4 above)
[the remark this answers is not in the document]
Comment (NikitaZotov) [reply sc-tools/sc-server/src/sc-server-impl/sc_server.hpp:21]: This const is surplus

File: sc-tools/sc-server/src/main.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (NikitaZotov) [reply sc-tools/sc-server/src/main.cpp:44]: This renamings are surplus

**Review by MksmOrlov — APPROVED** [review]

**Pull request merged**